.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,79 @@
+---
+name: Submit an issue
+about: Report a bug to help us improve
+title: "[Title]: Brief description"
+labels: ''
+assignees: ''
+
+---
+
+<!-- Please remove any unused sections.
+
+Note that anything written between these symbols will not appear in the actual, published issue. They serve as instructions for filling out this template. You may want to use the 'preview' tab above this textbox to verify formatting before submitting.
+
+Instructions:
+- Title line template: [Title]: Brief description
+
+-->
+
+## Actual behavior
+<!-- Briefly describe the current behavior, including screenshots or other references when applicable
+-->
+
+
+## Expected behavior
+<!-- Briefly describe the expected behavior (but that did not occur) -->
+
+
+## User-facing consequences
+<!--
+Implications and real-world consequences for learners, coaches, admins, and other users of the application
+-->
+
+
+### Errors and logs
+<!--
+Relevant logs from:
+ * the command line
+ * ~/.kolibri/logs/kolibri.txt
+ * the browser console
+
+Please wrap errors in triple backticks for clean formatting like this:
+```
+01:10 info: something happened
+01:12 error: something bad happened
+```
+-->
+
+
+## Additional information
+<!-- Include screenshots, code, or notes to help us better understand the issue -->
+### Traceback or relevant snippet from browser console
+<!--
```

Comment:
recommend avoiding nested sections. People are confused enough with HTML comments and markdown syntax – we definitely can't expect them to understand hierarchies thereof